ouroboros-consensus/src/Ouroboros/Consensus/Storage/LedgerDB/Update.hs

```diff
@@ -40,6 +40,7 @@ module Ouroboros.Consensus.Storage.LedgerDB.Update (
   , flush
   , prune
   , push
+  , pushMany
```

Comment:
Is it safe to export this? We need this to set up the dbchangelog in the mempool benchmarks

Reply:
I can't think of a reason why this would be unsafe. @Jasagredo ?